shoop/themes/classic_gray/templates/classic_gray/shoop/front/base.jinja

```diff
@@ -14,6 +14,8 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
     {# Include Styles #}
     <link rel="stylesheet" href="{{ STATIC_URL }}classic_gray/css/style.css">
+    {# Include all Extra Styles #}
+    {% block extrastyles %}{% endblock %}
```

Comment:
How is this being used?

There is also the add_resource context function in xtheme/resources.py which can insert JS or CSS into the head if you're just trying to add styles to a plugin. Not sure if that fits your needs.